enums.h

```diff
@@ -2871,3 +2871,9 @@ typedef enum action_id {
 	ACTION_SPELLPLR    = 25,
 	ACTION_SPELLWALL   = 26,
 } action_id;
+
+enum gold_quantity {
+  GOLD_SMALL_LIMIT = 1000,

```

Comment:
Bad indentation
```suggestion
	GOLD_SMALL_LIMIT = 1000,
```

Reply:
Weird. I didn't notice that when I removed the space at the end two lines down.